spec/factories/person.rb

```diff
@@ -9,6 +9,6 @@
     division2 'Command'
     title 'Captain'
     gender 'Female'
-    department 'Police'
+    association :department, factory: :department
```

Comment:
I would prefer to not put association in the factories. It tends to slow the tests, as you can end up creating the world from one object. Consider if department has `association : region`, now you're creating one of those too, when you just needed a person.